packages/inventory/src/ducks/inventory.ts

```diff
@@ -88,6 +89,7 @@ interface Filters {
  */
 const getIds = (store: Partial<Store>, props: Filters) => props.ids;
 const getServicePointIds = (store: Partial<Store>, props: Filters) => props.servicePointIds;
+const getActiveServicePoints = (store: Partial<Store>, props: Filters) => props.expired;
```

Comment:
`getActiveServicePoints` -> does this variable name correctly reflect the values its returning, or how its used?